packages/types/src/index.ts

```diff
@@ -659,3 +659,12 @@ export interface SimpleEvmOutput {
 export interface SimpleEvmBytecodeOutput {
     object: string;
 }
+
+export interface DutchAuctionDetails {
```

Comment:
Need a few changelog entries for:
* Types
* Migrations
* Contract wrappers
* Contract artifacts